quantmsrescore/annotator.py

```
@@ -0,0 +1,295 @@
+                    processes=self._processes,
+                )
+            except Exception as e:
+                logging.error(f"Failed to initialize MS2PIP: {str(e)}")
+                raise
+
+            psm_list = self._idxml_reader.psms
+            try:
+                ms2pip_generator.add_features(psm_list)
+                self._idxml_reader.psms = psm_list
+                logging.info("MS2PIP Annotations added to the PSMs")
+            except Ms2pipIncorrectModelException as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)